src/misc/rocm_smi_wrap.cc

```diff
@@ -42,6 +42,28 @@ ncclResult_t rocm_smi_init() {
   return ncclSuccess;
 }
 
```

Comment:
Please update copyright year for this file

Reply:
done